Samples~/LootLockerExamples/8 - Virtual Currency Purchases/LootLockerVirtualStore.cs

```diff
@@ -0,0 +1,126 @@
+using UnityEngine;
+using UnityEngine.UI;
+using LootLocker.Requests;
+
+namespace LootLocker
+{
+    public class LootLockerVirtualStore : MonoBehaviour
+    {
+        public static LootLockerVirtualStore Instance { get; private set; }
+
+        public Transform storeUI;
+        public Transform virtualPurchaseProductPrefab;
+
+        public Text goldAmount;
+
+        public string catalogKey = "";
+        public string walletID = "";
+
+        private string currencyCode;
+
+        public void Awake()
+        {
+
+            if (Instance == null)
+            {
+                Instance = this;
+            }
+            else Destroy(gameObject);
+
+            LootLockerSettingsOverrider.OverrideSettings();
+
+            LootLockerSDKManager.StartGuestSession((response) =>
+            {
+                if (!response.success)
+                {
+                    Debug.Log("Could not start guest session");
```

Comment:
```suggestion
                    Debug.Log("Could not start guest session, error: \""+response.errorData.ToString()+"\"");
```

Reply:
@AlmightyMikkel, why do you keep marking my comments as resolved but not actually either answering them or acting on them? This comment is still valid unless you disagreed.